torch/csrc/jit/passes/onnx/constant_fold.cpp

```diff
@@ -85,6 +85,9 @@ c10::optional<at::Tensor> runTorchSlice_opset9(const Node* node,
   std::vector<int64_t> axesAttr;
   if (node->hasAttributeS("axes")) {
     axesAttr = node->is(attr::axes);
+    for (auto& axis : axesAttr){
```

Comment:
nit : add a comment explaining that this is needed because opset 9 does not support negative indices?